subprojects/vinumc/vec.h

```diff
@@ -54,16 +67,18 @@
 		(vec)->capacity = 0;                                                               \
 	} while (0)
 
+/// Decrements the length of the vector by one.
 #define VEC_POP(arr)                                                                               \
 	do {                                                                                       \
 		(arr)->len--;                                                                      \
```

Comment:
I noticed this could cause an integer underflow if the length is already zero, which in turn could theoretically allow a buffer overflow attack. I'm going to check if `len > 0`.